utils/zerovec/derive/src/utils.rs

```diff
@@ -21,6 +21,26 @@ pub fn has_valid_repr(attrs: &[Attribute], predicate: impl Fn(&Ident) -> bool +
     })
 }
 
+pub fn has_transparent_or_cpacked_repr(attrs: &[Attribute]) -> bool {
```

Comment:
I couldn't figure out a way to make `has_valid_repr` work for us so I had to make a new thing. still planning on cleaning this up.